packages/allure-cypress/src/reporter.ts

```diff
@@ -489,19 +489,28 @@ export class AllureCypress {
   };
 }
 
-const getInitialSpecState = (): AllureSpecState => ({
+const createRuntimeState = (allureConfig?: AllureCypressConfig): AllureSpecState => ({
+  config: applyDefaultsToRuntimeConfig(allureConfig),
   initialized: false,
   messages: [],
   testPlan: parseTestPlan(),
 });
 
-/**
- * Explicitly enables the selective run feature.
- * @param config The Cypress configuration.
- */
-export const enableTestPlan = (config: Cypress.PluginConfigOptions) => {
-  config.env.allure = getInitialSpecState();
-  return config;
+const applyDefaultsToRuntimeConfig = ({
```

Comment:
The name of the function says that it returns the whole of the config, but modified one, but actually, it returns a single field. IMO `getConfigDefaults` would fit the logic much more, or let's change the logic to make the function return whole the config with applied defaults.

Reply:
I've renamed it to `getRuntimeConfigDefaults` so as not to confuse it with the user-exposed AllureCypressConfig instance.